modules/custom/openy_activity_finder/openy_cf_vue_app/dist/main.js

```diff
@@ -0,0 +1,12 @@
 or(y);y=y.parent}a(m)?w(0,[t],0,0):a(t.tag)&&b(t)}}return x(e,p,l),e.elm}a(t)&&b(t)}}({nodeOps:ir,modules:[wr,Tr,si,li,wi,K?{create:Ji,activate:Ji,remove:function(t,e){!0!==t.data.show?Vi(t,e):e()}}:{}].concat(yr)});Z&&document.addEventListener("selectionchange",function(){var t=document.activeElement;t&&t.vmodel&&na(t,"input")});var Qi={inserted:function(t,e,n,r){"select"===n.tag?(r.elm&&!r.elm._vOptions?ue(n,"postpatch",function(){Qi.componentUpdated(t,e,n)}):Yi(t,e,n.context),t._vOptions=[].map.call(t.options,Zi)):("textarea"===n.tag||nr(t.type))&&(t._vModifiers=e.modifiers,e.modifiers.lazy||(t.addEventListener("compositionstart",ta),t.addEventListener("compositionend",ea),t.addEventListener("change",ea),Z&&(t.vmodel=!0)))},componentUpdated:function(t,e,n){if("select"===n.tag){Yi(t,e,n.context);var r=t._vOptions,i=t._vOptions=[].map.call(t.options,Zi);if(i.some(function(t,e){return!R(t,r[e])}))(t.multiple?e.value.some(function(t){return Xi(t,i)}):e.value!==e.oldValue&&Xi(e.value,i))&&na(t,"change")}}};function Yi(t,e,n){Gi(t,e,n),(X||tt)&&setTimeout(function(){Gi(t,e,n)},0)}function Gi(t,e,n){var r=e.value,i=t.multiple;if(!i||Array.isArray(r)){for(var a,o,s=0,c=t.options.length;s<c;s++)if(o=t.options[s],i)a=D(r,Zi(o))>-1,o.selected!==a&&(o.selected=a);else if(R(Zi(o),r))return void(t.selectedIndex!==s&&(t.selectedIndex=s));i||(t.selectedIndex=-1)}}function Xi(t,e){return e.every(function(e){return!R(e,t)})}function Zi(t){return"_value"in t?t._value:t.value}function ta(t){t.target.composing=!0}function ea(t){t.target.composing&&(t.target.composing=!1,na(t.target,"input"))}function na(t,e){var n=document.createEvent("HTMLEvents");n.initEvent(e,!0,!0),t.dispatchEvent(n)}function ra(t){return!t.componentInstance||t.data&&t.data.transition?t:ra(t.componentInstance._vnode)}var ia={model:Qi,show:{bind:function(t,e,n){var r=e.value,i=(n=ra(n)).data&&n.data.transition,a=t.__vOriginalDisplay="none"===t.style.display?"":t.style.display;r&&i?(n.data.show=!0,Hi(n,function(){t.style.display=a})):t.style.display=r?a:"none"},update:function(t,e,n){var r=e.value;!r!=!e.oldValue&&((n=ra(n)).data&&n.data.transition?(n.data.show=!0,r?Hi(n,function(){t.style.display=t.__vOriginalDisplay}):Vi(n,function(){t.style.display="none"})):t.style.display=r?t.__vOriginalDisplay:"none")},unbind:function(t,e,n,r,i){i||(t.style.display=t.__vOriginalDisplay)}}},aa={name:String,appear:Boolean,css:Boolean,mode:String,type:String,enterClass:String,leaveClass:String,enterToClass:String,leaveToClass:String,enterActiveClass:String,leaveActiveClass:String,appearClass:String,appearActiveClass:String,appearToClass:String,duration:[Number,String,Object]};function oa(t){var e=t&&t.componentOptions;return e&&e.Ctor.options.abstract?oa(Ke(e.children)):t}function sa(t){var e={},n=t.$options;for(var r in n.propsData)e[r]=t[r];var i=n._parentListeners;for(var a in i)e[C(a)]=i[a];return e}function ca(t,e){if(/\d-keep-alive$/.test(e.tag))return t("keep-alive",{props:e.componentOptions.propsData
```

Comment:
Vue and Vue-router packaged here - please exclude them from this artifact.

Reply:
done